app/controllers/api/v1/admin/role_permissions_controller.rb

```diff
@@ -31,6 +32,19 @@ def update
         def role_params
           params.require(:role).permit(:role_id, :name, :value)
         end
+
+        def default_room
+          return unless role_params[:name] == 'CreateRoom' && role_params[:value] == true
+
```

Comment:
I orignally had this check as well here : 
```
role_permission = RolePermission.joins(:permission).find_by(role_id: role_params[:role_id], permission: { name: role_params[:name] })

return unless role_permission.value.to_s == 'false'
```
I removed it since technically there shouldn't be a case where the value for `CreateRooms` is already `True`and the person is sending a request to change it to `True`

Lmk if you think this extra check is needed to prevent a room being made when not needed @farhatahmad